src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -50,6 +50,21 @@ internal PublicClientApplicationBuilder WithUserTokenLegacyCachePersistenceForTe
             return this;
         }
 
+        /// <summary>
+        /// Configures the pca to use the recommended redirect Uri for the platform. 
```

Comment:
pca => Public client application.
redirect Uri => reply URI (to be aligned with the portal)